pkg/tfcleanup/resource_processor/btp_subaccount_subscription_processor.go

```diff
@@ -40,6 +41,20 @@ func addEntitlementDependency(body *hclwrite.Body, dependencyAddresses *generict
 
 		dependencyAddress := (*dependencyAddresses).EntitlementAddress[key]
 
+		if dependencyAddress == "" {
+			// Check if technical app name is different to service name/commercial app name
+			technicalAppName, commercialAppName, _ := btpcli.GetAppNamesBySubaccountAndApp(subaccountId, appName, btpClient)
+
```

Comment:
probably not break flow..but we could log an error if there are issues with the GetAppNamesBySubaccountAndApp right ?